scripts/yocto-build-and-test.sh

```diff
@@ -116,12 +124,12 @@ prepare_build_config() {
     local mender_artifact_version=$($WORKSPACE/integration/extra/release_tool.py --version-of mender-artifact --in-integration-version HEAD)
     local mender_connect_version=$($WORKSPACE/integration/extra/release_tool.py --version-of mender-connect --in-integration-version HEAD)
 
+    local sep="$(bitbake_override_separator)"
 
     local mender_binary_delta_version=$($WORKSPACE/mender-binary-delta/x86_64/mender-binary-delta --version | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
     cat >> $BUILDDIR/conf/local.conf <<EOF
-LICENSE_FLAGS_WHITELIST = "commercial_mender-binary-delta"
```

Comment:
I see. You will then add these into the `local.conf` for the different platforms in `dunfell` and `zeus`, right (similar to [this](https://github.com/mendersoftware/meta-mender/pull/1632))?

Reply:
Exactly.